.github/workflows/validate-theme-entry.yml

```diff
@@ -0,0 +1,269 @@
+                    if (!repository.data.has_issues) {
+                        addWarning('Your repository does not have issues enabled. Users will not be able to report bugs and request features.');
+                    }
+                } catch (e) {
+                    addError(`It seems like you made a typo in the repository field ${theme.repo}`);
+                    return;
+                }
+
+                let manifest = {};
+
+                try {
+                    let manifestFile = await github.rest.repos.getContent({
+                        owner,
+                        repo,
+                        path: 'manifest.json',
+                    });
+
+                    manifest = JSON.parse(Buffer.from(manifestFile.data.content, 'base64').toString('utf-8'));
+                } catch (e) {
+                    addError(`You don't have a valid \`manifest.json\` at the root of your repo.`);
+                    return;
+                }
+
+                let validManifestKeys = ['name', 'minAppVersion', 'author', 'version'];
+
+                for (let key in validManifestKeys) {
+                    if (!manifest.hasOwnProperty(key)) {
+                        addError(`Your manifest does not have the required \`${key}\` property.`);
+                    }
+                }
+
+                if (manifest.name !== theme.name) {
+                    addError(`Theme name mismatch, the name in this PR (\`${theme.name}\`) is not the same as the one in your repo (\`${manifest.name}\`). If you just changed your theme name, remember to change it in the manifest.json in your repo, and your latest GitHub release, if you have one.`);
+                }
+                if (manifest.authorUrl) {
+                  if (manifest.authorUrl === "https://obsidian.md") {
+                    addError(`The \`authorUrl\` field in your manifest should not point to the Obsidian Website. If you don't have a website you can just point it to your GitHub profile.`);
+                  }
+
+                  if (manifest.authorUrl.toLowerCase().includes("github.com/" + plugin.repo.toLowerCase())) {
+                    addWarning(`The \`authorUrl\` field in your manifest should not point to the GitHub repository of the plugin.`);
+                  }
+                }
+
+                if (manifest.fundingUrl && manifest.fundingUrl === "https://obsidian.md/pricing") {
+                  addError(`The \`fundingUrl\` field in your manifest should not point to the Obsidian Website, If you don't have a link were users can donate to you, you can just omit this.`);
+                }
+
+                if (!(/^[0-9.]+$/i.test(manifest.version))) {
+                    addError('Your latest version number is not valid. Only numbers and dots are allowed.');
+                }
+
+                if (/^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$/.test(theme.author)) {
```

Comment:
This should go further up to match up where we have it in the plugin validation script.